src/router.test.js

```diff
@@ -0,0 +1,3 @@
+const test = require('ava');
+
+test.todo('Write test');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
test.todo("Write test");
```